version/constants.go

```diff
@@ -132,15 +137,3 @@ func GetCompatibility(networkID uint32) Compatibility {
 		PrevMinimumUnmaskedVersion,
 	)
 }
-
-func GetLegacyCompatibility(networkID uint32) Compatibility {
```

Comment:
Will we need something like this when we go on to upgrade in future (e.g. `FlareHardFork2`)?

Reply:
No, because versions that nodes send will now always be aligned with the code version, so they have to upgrade the binary and restart the node anyway.